prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Hannah Moranda'; var numberOfPets = 3; var likesVegetables = 1; var nothing = null; let never;
+    console.log(fullName);
+    console.log(typeof 'Hannah Moranda');
+    console.log(numberOfPets);
+    console.log(typeof 3);
```

Comment:
all of the type of checks should be done with the variables, not the literal values